### PR TITLE
 Tensorflow model support 

### PR DESCRIPTION
**Pull request by Sakeebhasan123456**

### CODE REVIEW

**Review by ombhojane — CHANGES_REQUESTED** [review]
image
in utils, there are some functions are missing, that's why CI was failed.

also create a new file same as main.py, but this time, use tensorflow model imports instead of scikitlearn models we used earlier
in following snippets:
image

here scikit learn models has been used, same you have to use tensorflow models

here's the reference for tensorflow models: https://github.com/tensorflow/models/tree/master/official

**Comment by ombhojane** [comment]
example tf model (neural network models):
    model = Sequential([
        Dense(256, activation='relu', input_shape=input_shape),
        Dropout(0.3),
        Dense(128, activation='relu'),
        Dropout(0.3),
        Dense(64, activation='relu'),
        Dropout(0.3),
        Dense(32, activation='relu'),
        Dense(num_classes, activation='softmax')
    ])
    model.compile(optimizer='adam', loss='sparse_categorical_crossentropy', metrics=['accuracy'])
    return model

You may tryout with smaller models.

Note: have the previous scikitlearn support as it is, just in core.py add tensorflow models support, like when user use any tensorflow models in usage it'll be executable
